Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/DisplayMapperSRGB.azsl

```diff
@@ -36,7 +37,7 @@ PSOutput MainPS(VSOutput IN)
     PSOutput OUT;
 
     float3 color = PassSrg::m_framebuffer.Sample(PassSrg::LinearSampler, IN.m_texCoord).rgb;
-    OUT.m_color.rgb = pow(color, 1.0 / 2.2);
+    OUT.m_color.rgb = LinearToSRGB(AcesCg_To_LinearSrgb(color));
```

Comment:
I would prefer that we include Atom/Features/ColorManagement/TransformColor.azsli and do like this instead:
```
OUT.m_color.rgb = TransformColor(color, ColorSpaceId::ACEScg, ColorSpaceId::SRGB);
```

Then in the future when we make the working-color-space configurable it will use a shader option like this:
```
OUT.m_color.rgb = TransformColor(color, o_workingColorSpace, ColorSpaceId::SRGB);
```